### PR TITLE
Update Script #1

### PR DESCRIPTION
**Pull request by z0ne323**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.enabled": true
```
Comment (nigleweber) [inline .vscode/settings.json:2]: Nice!

File: script_vts_1.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            sys.exit(1)
+
+        return res
+
+    STRING = ""
+
+    path = get_path(STRING)
+
+    def read_api_key(file):
+        """Read API Key from the file path provided in get_path()"""
```
Comment (nigleweber) [inline script_vts_1.py:51]: What is this? Is it a comment??? I'm not familiar with this.

File: script_vts_1.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+    # Parse the argument
+    args = parser.parse_args()
+
+    #  TESTING CODE OR NO ? (might be good for the user to see the HASH?)
+    print("[+] Your SHA is: " + args.SHA)
+
+    # Sys.exit is good for handling error ? While loop until good result might be an option too...
+    # Might recode or put a helper function just before to list path when hitting tab...
+    def get_path(prompt):
```
Comment (nigleweber) [inline script_vts_1.py:24]: Let's move functions outside of main.

File: script_vts_1.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+import sys
+from pathlib import Path
+
+if __name__ == '__main__':
+
+    # Create the parser to have arguments in the script
+    parser = argparse.ArgumentParser()
+
+    # Adding SHA argument (let required True for now?)
+    parser.add_argument('--SHA', type=str, required=True)
```
Comment (nigleweber) [inline script_vts_1.py:14]: 👍 I think having it required is a good idea for now.
Let's look at adding a second flag for passing (multiple) SHAs in from a file later on.

File: script_vts_1.py [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+                if res.is_file():
+                    print("[+] We did find the file!")
+                else:
+                    print("[-] We didn't found the file!")
+                    sys.exit(1)
+            else:
+                print("[-] The path is not valid!")
+                sys.exit(1)
+        except ValueError:
+            sys.exit(1)
```
Comment (nigleweber) [inline script_vts_1.py:42]: You should be able to exit same as in the HelloWorld.py file. Saves you needing to import the sys library.
```exit(1) #Process exited with a failure```
Glad to see you exitted with an error status. :)

**Comment by nigleweber** [comment]
Great start man! I'll list out my notes:
1. Let's stick with the first iteration of the python script. Python lends itself to functions so let's not break the paradigm unless we have to by using classes. <- Super neat you did that in the other files btw
2. Move the functions outside (above) main
  - By using the main declaration ``` if __name__ == '__main__':``` at the bottom it forces python to load all other functions aka making them available as soon as main starts
3. Anytime you reach an exception let's try to use the error_handler I wrote. It'll point out the exact line where the error came from while also allowing you to add any additional debug messaging you want
  - You're welcome to change or add to my error_handler, but the ability to point to the line the error orignates from is super useful, strongly recommend keeping that logic in there


Rest of the code lgtm. 👍 
Nice work on the inline comments.

File: script_vts_1.py (hunk 2 above)
Comment (z0ne323) [reply script_vts_1.py:51]: It`s a comment, I don`t use it a lot but my linter told me I have to use this kind of comment not the #usual one like this 
to comment my functions but I can use #this comment outside of functions. (I follow the linter as much as possible like you said lol)

File: script_vts_1.py (hunk 4 above)
Comment (z0ne323) [reply script_vts_1.py:14]: adding it right now to the comment and working on it ASAP!

File: script_vts_1.py (hunk 5 above)
Comment (z0ne323) [reply script_vts_1.py:42]: Well, I know I'm always doing like you're doing, just calling exit(1) or 0 if successful, but my linter said no you have to be more precise you need to import sys and write sys.exit(1) to really tell that you're using this function from this module. (This practice is not really python related in itself it's also common for other languages to impose this when you learn c++ they're telling you never ever do idk for example ```using namespace std``` to simply call cin or cout you should always precise like std:::cout to sya that yeah you want cout function from std) So technically they're right it's a better practice to do it like this. But if you want, I have two solutions, first one I totally follow what you said only put exit no sys module OR I can limit the importation by doing something like from sys import exit or something like this?

Comment (nigleweber) [reply script_vts_1.py:42]: @v3r1t4s My way is more secure. As we discussed, we're probably not going to be writing code that needs more security so in this case I'd go with the linter. Also a #TIL moment for me too.

File: script_vts_1.py (hunk 2 above)
Comment (nigleweber) [reply script_vts_1.py:51]: Good call! The linter generally knows what's up.

**Pull request merged**